aptos-move/aptos-transaction-benchmarks/src/transactions.rs

```diff
@@ -131,9 +131,10 @@ impl TransactionBenchState {
         let new_block = BlockMetadata::new(
             HashValue::zero(),
             0,
-            1,
-            vec![],
+            0,
```

Comment:
just curious why 1 to 0?

Reply:
The round (0) and timestamp_usecs (1) remain the same. Note I changed the order of the arguments in BlockMetadata (moving timestamp_usecs to the end) to make it consistent with BlockMetadataTransaction, etc.